extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -46,31 +46,21 @@ const wrapper = {
   height: '100%',
   position: 'fixed',
   boxSizing: 'border-box',
-};
 
-// User overridable styles
-const defaultStyles = {
+  // User overridable styles
```

Comment:
Aren't all the props in this surrounding object overridable?

Reply:
Yeah that's a good point. I'll remove the comment.